mesh/ncmesh.hpp

```diff
@@ -421,6 +421,15 @@ class NCMesh
    int Geoms; ///< bit mask of element geometries present, see InitGeomFlags()
    bool Legacy; ///< true if the mesh was loaded from the legacy v1.1 format
 
+   static const int MaxElemNodes =
```

Comment:
I suggest making these `constexpr` and omitting the word "buffers" in the comments (that seems misleading to me, these are simply the numbers of element nodes/edges etc., nothing more)

Reply:
I guess constexpr is the more modern way of classwide constants like this?  Making this change...

Reply:
The goal is to make sure all the `for`-cycles in the `.cpp` have known bounds at compile-time, just like before...

Reply:
Ok the compiler doesn't like the constexpr approach so I went back to static const ints.